ZPastel.API/Controllers/PastelController.cs

```diff
@@ -40,5 +46,17 @@ public async Task<IReadOnlyCollection<PastelResource>> GetPasteis()
                 .Select(p => pastelConverter.ConvertToResource(p))
                 .ToList();
         }
+
+        [HttpPost("filter", Name = nameof(FilterPasteis))]
+        [Produces("application/json", Type = typeof(PageResource<PastelResource>))]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<ActionResult<PageResource<PastelResource>>> FilterPasteis(PastelFilterResource pastelFilterResource)
```

Comment:
This is not needed.

Reply:
Changed response to be IReadOnlyCollection<PastelResource>